tools/logger/logger.c

```diff
@@ -436,6 +436,11 @@ int main(int argc, char *argv[])
 	if (!config.in_file && !config.dump_ldc)
 		config.in_file = "/sys/kernel/debug/sof/etrace";
 
+	if (!config.in_file && baud) {
+		fprintf(stderr, "error: Input file not specified with -u option\n");
```

Comment:
```suggestion
		fprintf(stderr, "error: Input file not specified with -i option\n");
```

Reply:
@kv2019i  you thumbed up my comment and resolved it but made no change.

PS: thanks to eagle-eyed @ranj063 for spotting this.

Maybe even better:
```suggestion
		fprintf(stderr, "error: Input device not specified with -i option\n");
```

Reply:
@marc-hb  Sorry, I didn't you had a different issue. Ok, I'll rephrase that message.